main.css

```diff
@@ -0,0 +1,225 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap");
+.main-container {
+  height: 100%;
+  width: 100%;
+  background-color: white;
+  font-family: "Raleway", sans-serif;
+}
+
+header {
+  width: 70%;
+  background: url("./images/background.jpg") no-repeat;
+
```

Comment:
[Required] width of 70% did not allow you page to cover the whole screen. Please change it.